manifests/repo.pp

```diff
@@ -188,7 +188,7 @@
         logoutput => on_failure,
       }
 
-      if $::mrepo_env {
+      if $mrepo_env {
```

Comment:
Yeah, I think all the '' need to be switched to undefs, it's safe change as it would be the same regardless in versions of Puppet